command.go

```diff
@@ -230,6 +231,33 @@ func (c *Command) HelpFunc() func(*Command, []string) {
 	}
 }
 
+// Output the usage for the command
+// Used when a user provides invalid input
+// Can be defined by user by overriding UsageFunc
+func (c *Command) Usage() error {
+	c.mergePersistentFlags()
+	err := tmpl(c.getOutOrStderr(), c.UsageTemplate(), c)
```

Comment:
The problem is that `Usage()` is public, so anyone calling `Usage()` will not get the same result.  If `Usage()` were `usage()` that would be fine.